Makefile

```diff
@@ -139,10 +139,11 @@ fuzz-build:
 	rm -rf $(shell pwd)/build/fuzz/
 	mkdir -p $(shell pwd)/build/fuzz/out/
 
-	docker build . --tag local-fuzzing:latest -f tests/fuzz/Dockerfile.builder
+	docker build . --pull --tag local-fuzzing:latest -f tests/fuzz/Dockerfile.builder
 	docker run --rm \
```

Comment:
By switching to native fuzz I'm wondering do we still need this image?

Reply:
Great question! When this project moves to native fuzz we will need to keep this or find a better way to ensure we are not breaking the [oss-fuzz](https://github.com/google/oss-fuzz) builds upstream.

One of the reasons we use our approach is to have a "fail-fast" repository-specific test. We could drop this off and rely on the [ci-fuzz action](https://google.github.io/oss-fuzz/getting-started/continuous-integration/), which will assess [all our repositories instead](https://github.com/google/oss-fuzz/blob/master/projects/fluxcd/Dockerfile#L25-L26).

I think we can reconsider that once we have all repos migrated to native fuzz. The only issue is that it will be a steeper learning curve for folks to find issues and mitigate them, specially when they work for native fuzz, but break on oss-fuzz. Another point to take into account will be the time taken on cifuzz and the flakiness it may bring, as I am not sure we will be able to make it repo-specific.